Scenarios/AKS-Secure-Baseline-PrivateCluster/Apps/SimpleGMSAApp/Getting-Started.md

```diff
@@ -62,40 +65,69 @@ Follow the steps [here](https://learn.microsoft.com/virtualization/windowscontai
 1. If you are running these commands on your domain controller that is a Windows Server machine, you may have trouble
 installing the [Kubernetes CLI (kubectl)](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/) and [kubelogin](https://github.com/Azure/kubelogin). The error would mention "Install-AzAksKubectl". If so, you will need to manually install both following the links above.
 
-## How to Validate Your GMSA Integration
+## How to Validate Your GMSA Integration before Deployment
+
+1. To validate that your cluster is successfully retrieving your GMSA, go into your domain controller local server menu, go to Tools and select Event Viewer. Look under ActiveDirectory events. Look at the contents of the most recent events for a message that says "A caller successfully fetched the password of a group managed service account." The IP address of the caller should match one of your AKS cluster IPs.
+
+## Import Simple Application Container Image
```

Comment:
```suggestion
## Import simple GSMA application to your container registry
```

Reply:
Shoot, had a typo in there, and fixed it just as you were merging it in.  Sorry!